lib/utils/flip-object.ts

```diff
@@ -0,0 +1,18 @@
+/** Transform object keys into values and vice-versa */
+export const flipObject = <O extends Record<string, string>>(
```

Comment:
This is cool but since lodash is installed, we can use https://lodash.com/docs/4.17.15#invert instead of a custom helper.